src/providers/v3/subgraph-provider.ts

```diff
@@ -59,6 +59,8 @@ const SUBGRAPH_URL_BY_CHAIN: { [chainId in ChainId]?: string } = {
     'https://api.thegraph.com/subgraphs/name/jesse-sawa/uniswap-celo',
   [ChainId.GÖRLI]:
     'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v3-gorli',
+  [ChainId.BSC]:
+    'https://api.thegraph.com/subgraphs/name/ilyamk/uniswap-v3---bnb-chain',
```

Comment:
double checking we know/trust this subgraph? Might be preferable to have one run by labs

Reply:
since owner can update in place

Reply:
Hmm honestly we may want to just use the static subgraph provider for now until we can get one up ourselves: https://github.com/Uniswap/smart-order-router/blob/main/src/providers/v3/static-subgraph-provider.ts

Reply:
Tina and team are trying to get one up that's run by labs today, so we can update it here when done